scripts/scil_filter_tractogram.py

```diff
@@ -158,11 +158,8 @@ def main():
                              'not compatible.')
 
             mask = img.get_data()
-            filtered_streamlines, indexes = filter_grid_roi(
-                sft,
-                mask,
-                filter_mode,
-                is_not)
+            filtered_sft, indexes = filter_grid_roi(sft, mask,
+                                                    filter_mode, is_not)
 
         elif filter_type == 'atlas_roi':
             img = nib.load(filter_arg_1)
```

Comment:
Would this be more intuitive and less repetitive? (changing filter_arg_1 for filter_arg line 148)

```
        if filter_type == 'drawn_roi' or filter_type=='atlas_roi':
            img = nib.load(filter_arg)
            if not is_header_compatible(img, sft):
                parser.error('Headers from the tractogram and the mask are '
                             'not compatible.')
            if filter_type == 'drawn_roi':
                mask = img.get_data()
            else:
                atlas = img.get_data().astype(np.uint16)
                mask = np.zeros(atlas.shape, dtype=np.uint16)
                mask[atlas == int(filter_arg_2)] = 1
            filtered_sft, indexes = filter_grid_roi(sft, mask,
                                                    filter_mode, is_not)
```